packages/integration/spec/templates/default-standard-user-workflow/application.json

```diff
@@ -16,7 +16,7 @@
     }
   },
   "mailjet": {
-    "apikey-public": "f81f5b1d64a395a528e5f5c44aeba1af",
-    "apikey-private": "445647e5cf8517969b95e3363c3db7d2"
+    "apikey-public": "abb13e7529f0bb01e9768a8068017556",
```

Comment:
Keys should not be committed in sources. We should provide them externally (from given for example). We should also be able to override values using environment variables (for Jenkins)